dist/gmusic.js

```diff
@@ -239,6 +245,87 @@ proto.playback = {
   }
 };
 
+// Create a queue API
+proto.queue = {
+  _fireEvent: function (el, etype) {
+    if (el.fireEvent) {
+      el.fireEvent('on' + etype);
```

Comment:
`fireEvent` isn't well documented, please ditch using it.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/fireEvent